webapp/src/components/input.jsx

```diff
@@ -77,6 +77,7 @@ export default class Input extends PureComponent {
                     id={this.props.id}
                     className='form-control'
                     rows='5'
+                    readOnly
```

Comment:
Needs to be:
```suggestion
                    readOnly={true}
```
to make the linter happy.